crates/agglayer-config/src/l1.rs

```diff
@@ -57,6 +63,10 @@ fn default_ws_node_url() -> Url {
     "ws://zkevm-mock-l1-network:8546".parse().unwrap()
 }
 
-const fn default_max_reconnection_elapsed_time() -> Duration {
+const fn default_total_reconnect_timeout() -> Duration {
+    Duration::from_secs(10 * 60)
+}
+
+const fn default_reconnect_attempt_interval() -> Duration {
     Duration::from_secs(10)
```

Comment:
Nit: I wonder if this should be higher, as we are already using exponential scale starting from default initial 500 ms until this value?